developer/src/kmc-copy/src/copier-messages.ts

```diff
@@ -96,20 +96,117 @@ export class CopierMessages {
 
   static ERROR_UnsupportedProjectVersion = SevError | 0x000D;
   static Error_UnsupportedProjectVersion = (o:{filename: string, version: string}) => m(this.ERROR_UnsupportedProjectVersion,
-    `Project version ${def(o.version)} for '${def(o.filename)}' is not supported by this version of Keyman Developer`);
+    `Project version ${def(o.version)} for '${def(o.filename)}' is not supported by this version of Keyman Developer`
+  );
 
   static ERROR_InvalidProjectFile = SevError | 0x000E;
   static Error_InvalidProjectFile = (o:{filename: string, message: string}) => m(
     this.ERROR_InvalidProjectFile,
-    `Project file '${def(o.filename)}' is not valid: ${def(o.message)}`);
+    `Project file '${def(o.filename)}' is not valid: ${def(o.message)}`
+  );
 
   static ERROR_ProjectFileCouldNotBeRead = SevError | 0x000F;
   static Error_ProjectFileCouldNotBeRead = (o:{filename: string}) => m(
     this.ERROR_ProjectFileCouldNotBeRead,
-    `Project file '${def(o.filename)}' could not be read`);
+    `Project file '${def(o.filename)}' could not be read`
+  );
 
   static INFO_DryRun = SevInfo | 0x0010;
   static Info_DryRun = (o:{outPath: string}) => m(
     this.INFO_DryRun,
-    `Dry run requested. No changes have been saved`);
+    `Dry run requested. No changes have been saved`
+  );
+
+  static ERROR_InvalidGitHubSource = SevError | 0x0011;
+  static Error_InvalidGitHubSource = (o:{source: string}) => m(
+    this.ERROR_InvalidGitHubSource,
+    `Source project specification '${def(o.source)}' is not a valid GitHub reference`,
+    `The source project specification for GitHub sources must match the pattern:
+        github:\\<owner/repo>[:\\<branch>]:\\<path>
+    The path must include the .kpj filename and may optionally begin with a forward slash.
+    The following are valid examples:
+        github:keymanapp/keyboards:master:release/k/khmer_angkor/khmer_angkor.kpj
```

Comment:
Is the pattern from l.125 expecting slashes?
github:\\<owner/repo>...?

Reply:
The backslashes are due to markdown escaping, I am not 100% sure they are required